protocols/identify/src/handler.rs

```diff
@@ -255,6 +302,46 @@ impl ConnectionHandler for Handler {
             }
         }
 
+        // Check for pending replies to send.
+        if let Some(ref info) = self.info {
```

Comment:
NB: I thought `let_chains` had landed already been stabilized, but[ it was reverted](https://github.com/rust-lang/rust/pull/100538).

Reply:
yeah, compiler points me to https://github.com/rust-lang/rust/issues/53667, see https://github.com/rust-lang/rust/issues/53667#issuecomment-1297933589